### PR TITLE
Add Tests for worker_util.py

### PR DESCRIPTION
**Pull request by nsjcorps**
I have added tests for the worker_util which is used by the RL-worker to communicate with EvalAI Server to evaluate Docker Based Challenge submissions

### CODE REVIEW

**Comment by nsjcorps** [comment]
I have finished writing tests for most of the file. However, I do not know what the response of the EvalAI server is supposed to be. Eg. I do not know what the get_message_from_sqs_queue function returns from the server. How can I find out what they actually are? @utsav @vkartik97 @RishabhJain2018 @Ram81

**Comment by vkartik97** [comment]
> I do not know what the get_message_from_sqs_queue function returns from the server.

You can try using `Postman` to get responses of all the endpoints used in EvalAI Interface.

**Comment by nsjcorps** [comment]
@vkartik97 As mentioned before sir, I can't use docker. So I won't be able to start the server. Is there anyway I can get the expected response sir?

**Comment by vkartik97** [comment]
You can have a look at view which is endpoints for : /api/jobs/challenge/queues/{QUEUE_NAME}

**Comment by nsjcorps** [comment]
I did. But it doesn't show me what the json would look like @vkartik97

**Comment by nsjcorps** [comment]
It only says that it returns some data from the database @vkartik97

**Comment by nsjcorps** [comment]
I don know what the data from the database looks like @vkartik97

**Comment by nsjcorps** [comment]
Never mind sir. I have found the way to test the function. Thank you very much  @vkartik97

**Comment by nsjcorps** [comment]
@vkartik97 I have written the test cases after a load of commits ;)...Please take a look

File: tests/integration/worker/test_worker_util.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    def test_return_url_per_environment(self):
+        expected = self.url_per_environment
+        result = self.evalai_interface.return_url_per_environment(self.test_url)
+        self.assertEqual(expected, result)
+
+
+@mock.patch("scripts.workers.worker_util.EvalAI_Interface.return_url_per_environment")
+@mock.patch("scripts.workers.worker_util.EvalAI_Interface.make_request")
+class APICallsTestClass(BaseAPITestClass):
+
```
Comment (pushkalkatara) [inline tests/integration/worker/test_worker_util.py]: please remove this extra space.

File: tests/integration/worker/test_worker_util.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import mock
+
+from rest_framework.test import APITestCase
+from scripts.workers.worker_util import EvalAI_Interface, URLS
+
+
+AUTH_TOKEN = "mock_auth_token123456"
```
Comment (pushkalkatara) [inline tests/integration/worker/test_worker_util.py]: we can define these variables in setUp itself.

File: tests/integration/worker/test_worker_util.py (hunk 1 above)
Comment (nsjcorps) [reply tests/integration/worker/test_worker_util.py]: done

**Comment by pushkalkatara** [comment]
@vkartik97 Looks good to me. We can merge this and approve the GCI task.

**Comment by nsjcorps** [comment]
Thank you @pushkalkatara

File: tests/integration/worker/test_worker_util.py (hunk 2 above)
Comment (nsjcorps) [reply tests/integration/worker/test_worker_util.py]: Done!! @pushkalkatara

**Pull request closed without merging**